server/plugin/utils_test.go

```diff
@@ -694,9 +703,79 @@ func TestParseSubscriptionsToCommandResponse(t *testing.T) {
 			subscriptionsList: []*serializers.SubscriptionDetails{},
 			expectedMessage:   fmt.Sprintf("No %s subscription exists", constants.CommandBoards),
 		},
+		{
+			description:       "ParseSubscriptionsToCommandResponse: error in fetching filtered subscription list",
+			command:           constants.CommandBoards,
+			subscriptionsList: []*serializers.SubscriptionDetails{},
+			expectedMessage:   "Something went wrong, please try again later",
+			err:               errors.New("error in fetching filtered subscription list"),
+		},
+		{
+			description: "ParseSubscriptionsToCommandResponse: subscriptions created by the user",
+			command:     constants.CommandBoards,
+			subscriptionsList: []*serializers.SubscriptionDetails{
+				{
+					ChannelID:        "mockChannelID",
+					MattermostUserID: "mockUserID",
+					ServiceType:      constants.CommandBoards,
+					SubscriptionID:   "mockSubscriptionID",
+					OrganizationName: "mockOrganizationName",
+					ProjectName:      "mockProjectName",
+					EventType:        constants.SubscriptionEventWorkItemCreated,
+					CreatedBy:        "mockCreatedBy",
+					ChannelName:      "mockChannelName",
+				},
+			},
+			createdBy:       constants.FilterCreatedByMe,
+			expectedMessage: fmt.Sprintf("###### %s subscription(s)\n| Subscription ID | Organization | Project | Event Type | Created By | Channel |\n| :-------------- | :----------- | :------ | :--------- | :--------- | :------ |\n| mockSubscriptionID | mockOrganizationName | mockProjectName | Work Item Created | mockCreatedBy | mockChannelName |\n", cases.Title(language.Und).String(constants.CommandBoards)),
```

Comment:
Same.

Reply:
No constants present for this.....this string is printed in three different parts.....i combined them into one